flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/PointwiseBlockingResultInfo.java

```diff
@@ -60,4 +61,30 @@ public long getNumBytesProduced() {
                 .flatMapToLong(Arrays::stream)
                 .reduce(0L, Long::sum);
     }
+
+    @Override
+    public long getNumBytesProduced(
+            IndexRange partitionIndexRange, IndexRange subpartitionIndexRange) {
+        long inputBytes = 0;
+        for (int i = partitionIndexRange.getStartIndex();
+                i <= partitionIndexRange.getEndIndex();
+                ++i) {
+            checkState(
+                    subpartitionBytesByPartitionIndex.get(i) != null,
+                    "Partition index %d is not ready.",
+                    i);
+            checkState(
+                    subpartitionIndexRange.getEndIndex()
+                            < subpartitionBytesByPartitionIndex.get(i).length,
+                    "Subpartition end index %d is out of range of partition %d.",
```

Comment:
We should use `%s` here because the [checkState](https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/util/Preconditions.java#:~:text=throw%20new%20IllegalStateException(format(errorMessageTemplate%2C%20errorMessageArgs))%3B) can't parse `%d`

Reply:
Thanks for your remind! Fix this.